andela/activo-web
# #159522992 Users can download asset categories as csv

**What does this PR do?**
Implements the functionality that enables a user to download the asset categories in csv format


**Description of Task to be completed?**
- create action creators to handle client-side request to download asset-categories
- implement logic to download file as csv
- write tests to cover implementation

**How should this be manually tested?**
After cloning the application, navigate to the dashboard and click on the `export` button. The download should begin and on successful download, the csv file should be saved to local memory.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#159522992](https://www.pivotaltracker.com/story/show/159522992)

**Screenshots (if appropriate)**
<img width="1440" alt="screen shot 2018-08-04 at 3 41 51 pm" src="https://user-images.githubusercontent.com/24798364/43677629-f2467320-97fc-11e8-9ee7-3f4280cb7bc2.png">


**Questions:**
N/A
